### ISSUE TITLE
Input accumulation is off by default but the documentation states it is on by default

### ISSUE THREAD

**Issue by m4nu3lf** [godotengine/godot#55037]
### Godot version

da5c843bd1ba7bce5d56bd9b8dd13fb1a1983c96

### System information

OS: Arch Linux, CPU: Ryzen 5, GUP: RX 480

### Issue description

The documentation states that input accumulation is enabled by default, but I'm pretty sure it isn't. I get many InputMouseMotion events every frame unless I explicitly call `Input.set_use_accumualted_input(true)`.

Should we change the docs or the implementation?

### Steps to reproduce

Create a scene with a node and attach a script. Add something like this:

```
func _input(event: InputEvent) -> void:
  if event is InputEventMouseMotion:
     print(Engine.get_drawn_frames())
```

You should see the same frame number show up multiple times.

**Comment by Calinou** [comment]
I'm pretty sure input accumulation is supposed to be enabled by default. Can you reproduce this in 3.3?

**Comment by m4nu3lf** [comment]
I can reproduce it with the latest commit 45088780be124acc8ab226d3e5cee808bb626980
This is the code:
```
extends Spatial

func _ready() -> void:
	Input.set_use_accumulated_input(true)

func _input(event) -> void:
	if event is InputEventMouseMotion:
		print(Engine.get_frames_drawn())

```
Try to comment uncomment the _ready() method to see the issue.

**Comment by Calinou** [comment]
See also https://github.com/godotengine/godot/issues/54066.

**Comment by akien-mga** [comment]
`main/input_default.cpp` has `use_input_accumulation` set to `false`, so it's indeed disabled by default. It used to be `true` but this was changed in #42220. It is however enabled by default in the editor (`editor/editor_node.cpp` sets it on).

I don't know if making it off-by-default was intentional. If so, maybe the documentation simply needs to be updated. What part of the documentation says that it's enabled by default?

**Comment by akien-mga** [comment]
Yeah given that this was changed in 3.4, users are now used to the new default value, even if it wasn't intended. So I would keep it as is in 3.5, it's too late now to revert back (there might be bugs with input accumulation that we haven't caught during beta phase due to it being off by default).

So documenting it should be enough indeed. We could consider turning it back on for 3.6 with some extensive testing.

**Comment by akien-mga** [comment]
Given the amount of regression reports we got which ended up due to input accumulation being mistakenly turned off in 3.4, I've been thinking about turning it back on for 3.5, even if it's quite late to do so now as discussed above.

Notably:
- #54531
- #54909 
- #57039 
- #57386 
- #57599

We'll have 3.5 RC 6 to try it out, which should have at least 10 days of testing as I'll be away on holiday :)
So we can assess whether testers seem to be fine with the change or not. And then eventually either flip it back to `false` if we find new issues with input accumulation and prefer to go back to the 3.4 behavior, or keep it `true` to get back the 3.1 - 3.3 behavior and (when relevant) improved input handling performance.

**Comment by akien-mga** [comment]
Fixed by #62826.
